### ISSUE TITLE
How to install and ship it

### ISSUE THREAD

**Issue by AstraLuma** [alexrochas/i3-workspace-rollback#1]
1. Write a `setup.py`
2. Upload to PyPI with [twine](https://github.com/pypa/twine)
3. ???
4. Profit!

**Comment by alexrochas** [comment]
Hi, thanks for reply.

Yeah, yesterday I was thinking not exactly how to ship, but how to add to i3.
Your suggestion is what I was thinking. So, my conclusion is that in "step 3" you will need, after install from PyPI, add to your i3 config something like:

```
exec i3-workspace-rollback
```

This if I add a script to my python package. Makes sense? What do you think?
Thanks again!

**Comment by AstraLuma** [comment]
Yup! Add a `__main__.py` and [have setuptools handle it](https://chriswarrick.com/blog/2014/09/15/python-apps-the-right-way-entry_points-and-scripts/).

**Comment by AstraLuma** [comment]
And then users would be: 
1. `pip install --user i3-workspace-rollback`
2. Adjust `.i3/config` to taste
